tools/language-server/logstat/src/main.rs

```diff
@@ -330,7 +330,7 @@ async fn main() -> Result<()> {
                 let first_operation = &iteration.operations[0];
                 &first_operation.timestamp < start_time
             })
-            .collect_vec();
```

Comment:
Changes in LS tools caused by removal of `itertools` from prelude. That was one of the last remaining usages, and it was trivial to replace.